sound/soc/sof/pm.c

```diff
@@ -465,7 +465,7 @@ int snd_sof_suspend(struct device *dev)
 	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
 	int ret;
 
-	if (sdev->s0_suspend) {
+	if (snd_sof_dsp_d0i3_suspend(sdev)) {
```

Comment:
the changes look odd.

in the resume() you call a suspend() operation, which means the naming isn't quite right.

snd_sof_dsp_d0i3_used() would make more sense to me.

Reply:
How about snd_sof_is_d0i3_suspend(), @plbossart ?

Reply:
it's also confusing to use  snd_sof_is_d0i3_suspend() in a suspend() routine.

maybe snd_sof_dsp_d0i3_on_suspend() ?

Reply:
> it's also confusing to use snd_sof_is_d0i3_suspend() in a suspend() routine.
> 
> maybe snd_sof_dsp_d0i3_on_suspend() ?

Sounds good, changing it now.